theforeman.org/yaml/views/release.yml

```diff
@@ -15,6 +15,7 @@
         - '8'
```

Comment:
Unrelated, but we should clean up 7 & 8 here. Looks like they don't show up in the UI, so it doesn't matter now, but still nice to clean up.